docs/src/components/templates/FrameworkSwitcher.tsx

```diff
@@ -18,10 +18,10 @@ const FrameworkSwitcher: React.FC<Props> = () => {
 
   function onChange(event: RadioChangeEvent) {
     const framework = event.target.value;
-
+    console.log('env', process.env);
```

Comment:
We can probably remove this line?

Reply:
Yeah, thank you! I did it in [this](https://github.com/cube-js/cube.js/pull/3362/commits/4d2f6b96b61db8487a7a7e9f8391686bd17f4eb6) commit and fix the framework switcher! Now it's ready for production I guess.